### ISSUE TITLE
Tutorial Guide

### ISSUE THREAD

**Issue by jaggu21** [Canvasbird/canvasboard#243]
**Is your feature request related to a problem? Please describe.**
Adding tutorial guide to the board would be very helpful for the first time users.   

**Describe the solution you'd like** 💡
We can use a javascript library called `Driver.js` which can drive the user's focus across the page. It has great features and is easy to work with.This is the [Documentaion](https://kamranahmed.info/driver.js/).

**Comment by goliakshay357** [comment]
This would be a great one to work on.

**Comment by kanishkaa24** [comment]
Can you elaborate on this issue ?

**Comment by jaggu21** [comment]
@kanishkaa24 
Tutorial Guide will elaborate the components present on the page. Here is an example from [Circuitverse](https://circuitverse.org/simulator). I hope this helps!
![sc](https://user-images.githubusercontent.com/56889104/103204705-f4826e80-491d-11eb-8c06-37bcb78fb14f.png)

**Comment by shraddha-1402** [comment]
I would like to work on this issue! Please assign me this!

**Comment by goliakshay357** [comment]
Yeah sure! Looking forward to a PR :smiley: !

**Comment by shraddha-1402** [comment]
hey @goliakshay357 how can i make changes to the board without login? are there any test credentials?

**Comment by goliakshay357** [comment]
Hello @shraddha-1402,
Can you please describe more?

You can use your own login credentials, Should not be a problem. 
Thanks!

**Comment by shraddha-1402** [comment]
> Hello @shraddha-1402,
> Can you please describe more?
> 
> You can use your own login credentials, Should not be a problem. Please feel free to join out slack channel.
> Thanks!

I wanted to know that i'll have to implement activity driver in board section, right? 
And for that, is there a way i can make changes in board section without logging in?

**Comment by Kajol-Kumari** [comment]
Hey @shraddha-1402 you can't access the board without logging in so you need to first signup with your email ID , username and password. Then verify your email with the link that will be sent on your emailID and after than you can just login and access all the workspaces along with boards. 

Let me know if you need further clarification

**Comment by shraddha-1402** [comment]
> Hey @shraddha-1402 you can't access the board without logging in so you need to first signup with your email ID , username and password. Then verify your email with the link that will be sent on your emailID and after than you can just login and access all the workspaces along with boards.
> 
> Let me know if you need further clarification

Okay @Kajol-Kumari. Thank you!

**Comment by goliakshay357** [comment]
@shraddha-1402,
Are you working on this issue?

Thanks.

**Comment by shraddha-1402** [comment]
> @shraddha-1402,
> Are you working on this issue?
> 
> Thanks.

Yes @goliakshay357 . I am new to typescript so i am learning while doing.
